src/ui/components/buttons/externalLink/ExternalLink.web.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { openUrl } from 'features/navigation/helpers'
+import { extractExternalLinkParts } from 'ui/components/buttons/externalLink/ExternalLink.service'
+import { ExternalSite } from 'ui/svg/icons/ExternalSite'
+import { ColorsEnum, Spacer, Typo } from 'ui/theme'
+
+interface Props {
+  url: string
+  text?: string
+  color?: ColorsEnum
+  testID?: string
+}
+
+export const ExternalLink: React.FC<Props> = ({ url, text, color, testID }) => {
+  text = text || url
+
+  const [firstWord, remainingWords] = extractExternalLinkParts(text)
```

Comment:
```suggestion
  const [firstWord, remainingWords] = extractExternalLinkParts(text || url)
```

étonnant de modifier les props à la volée. tu peux changer dans l'autre aussi ?

Reply:
bien vu, super louche ! j'ai changé ça